releasenotes/notes/45489.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+
+issue:
+  - 45338
+
+releaseNotes:
+- |
+   **Fixed** ExcludeIPRanges configuration not producing expected iptables rules when given ranges contains space after comma separator
```

Comment:
I think this description is not related to what you have been fixed?

Reply:
already edited